.goreleaser-nightly.yml

```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+version: 2
+pro: true
+
+release:
+  name_template: 'v{{.Version}}'
+
+
```

Comment:
```suggestion

```

Reply:
cannot apply on deleted lines :(
I'll do it